### PR TITLE
Add nginx-app in fleet-test-data for QA automation

### PR DESCRIPTION
**Pull request by sbulage**
We are using fleet-test-data repository in automation, so we require few application to be present here. Now starting with Nginx. As we grow with automation will add more applications for testing purpose.

### CODE REVIEW

File: qa-test-apps/nginx-app/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# In this example, we are using fleet option keepResources: true/false.
```
Comment (manno) [inline qa-test-apps/nginx-app/README.md:1]: I wonder if we should capitalize Fleet 

```suggestion
# In this example, we are using the Fleet option keepResources: true/false.
```

File: qa-test-apps/nginx-app/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# This option will ensure that deployed application will not be deleted
+# after deleting GitRepo.
```
Comment (manno) [inline qa-test-apps/nginx-app/README.md:3]:
```suggestion
# after deleting a GitRepo.
```

File: qa-test-apps/nginx-app/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  - qa-test-apps/nginx-app
+  targetNamespace: keep-resources
+  keepResources: true
+  targets:
+    - clusterSelector:
+        matchExpressions:
+          - key: provider.cattle.io
+            operator: NotIn
+            values:
+              - harvester
```
Comment (manno) [inline qa-test-apps/nginx-app/README.md:26]: That should also deploy to all clusters (https://fleet.rancher.io/gitrepo-targets#target-matching).
```suggestion
  targets:
    - clusterSelector: {}
```

File: qa-test-apps/nginx-app/README.md (hunk 1 above)
Comment (sbulage) [reply qa-test-apps/nginx-app/README.md:1]: Sure I will do that.

**Pull request merged**